src/lib/objects/hello-world/hello-world-receiver.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { page } from '$app/stores'
+	import adapter from '$lib/adapters'
+	import Button from '$lib/components/button.svelte'
+	import type { DataMessage } from '$lib/stores/chat'
+	import { objectKey, objectStore } from '$lib/stores/objects'
+	import { profile } from '$lib/stores/profile'
+	import { walletStore } from '$lib/stores/wallet'
+	export let message: DataMessage
+
+	$: name = (
+		$objectStore.objects.get(objectKey(message.objectId, message.instanceId)) as { name?: string }
+	)?.name
```

Comment:
On a call we talked about passing these things in as props. I am fine to merge it as is, just want to make sure you have not forgot about it and this is here left deliberately.

Reply:
Indeed, I will merge this and then I will work on passing props as the next step.